src/screens/timeStamping/ManualTimeStamping/index.tsx

```diff
@@ -89,6 +91,9 @@ const ManualTimeStamping = ({ route }: ManualTimeStampingProps) => {
       </ScrollView>
       <NiPrimaryButton title='Valider et horodater' style={styles.submitButton} onPress={timeStampEvent}
         loading={loading} />
+      <TouchableOpacity onPress={() => goToQRCodeScanner()}>
```

Comment:
ici tu peux ajouter `hitSlop={{ top: 12, bottom: 12 }}` pour augmenter la zone de clic -> _Sets additional distance outside of element in which a press can be detected._ d'apres la doc

tu pourrais meme definir ca dans les constante `styles/metrics`
et l'ajouter aussi sur le texte de la page du QRCode
et modifier la ou `hitslop` est deja utilisé i.e. sur la page d'authentification

Reply:
est ce que `onPress={goToQRCodeScanner}` fonctionne ?